.gitignore

```diff
@@ -0,0 +1,4 @@
+bower_components
+node_modules
+npm-debug.log
+bedrock
```

Comment:
Let's not ignore 'bedrock' for now. We may add it in later. We're currently trying to figure out whether or not we should ignore it or not.

For right now, remove bedrock from .gitignore.